lib/phoenix_component/declarative.ex

```diff
@@ -904,17 +903,18 @@ defmodule Phoenix.Component.Declarative do
   end
 
   defp build_attr_doc_and_default(%{doc: doc, type: :global, opts: opts}, indent) do
```

Comment:
But, clearly, at some point, we intended to call this function for the global type.